### PR TITLE
AnimeFenix without changing other files

### PR DESCRIPTION
**Pull request by Keiser04**
In local I used to get the test but after repeating it so much, as soon as the new link opens, it closes. it doesn't wait although animefenix has a capcha it seems
image

### CODE REVIEW

File: src/pages/AnimeFenix/meta.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "urls": {
+    "search": "https://www3.animefenix.tv/animes",
+    "match": ["*://animefenix.tv/*"]
```
Comment (lolamtisch) [inline src/pages/AnimeFenix/meta.json:4]: Does not match anymore needs to be *://*.ani

File: src/pages/AnimeFenix/main.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    },
+    getOverviewUrl: url => {
+      const urlParts = url.split('/');
+      if (urlParts[3] === 'ver') {
+        urlParts.splice(3, 1);
+      }
+      urlParts[urlParts.length - 1] = urlParts[urlParts.length - 1].replace(/-\d+$/, '');
+      return urlParts.join('/');
+    },    
+    getEpisode: url => {
```
Comment (lolamtisch) [inline src/pages/AnimeFenix/main.ts:39]: Episode wrong: 
https://www3.animefenix.tv/ver/kono-yo-no-hate-de-koi-wo-utau-shoujo-yu-no-26.5
https://www3.animefenix.tv/ver/kusuriya-no-hitorigoto-22?page=232323

**Review by lolamtisch — CHANGES_REQUESTED** [review]

**Comment by Keiser04** [comment]
nvm now is not detecting the web

File: src/pages/AnimeFenix/main.ts (hunk 2 above)
Comment (Keiser04) [reply src/pages/AnimeFenix/main.ts:39]: i need to let the 26.5 or is correct if is 26?

Comment (Keiser04) [reply src/pages/AnimeFenix/main.ts:35]: @lolamtisch

File: src/pages/AnimeFenix/main.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  name: 'AnimeFenix',
+  domain: 'https://animefenix.tv',
+  languages: ['Spanish'],
+  type: 'anime',
+  isSyncPage: url => {
+    const path = url.split('/')[3];
+    return path === 'ver';
+  },
+  sync: {
+    getTitle: url => {
```
[the remark this answers is not in the document]
Comment (Keiser04) [reply src/pages/AnimeFenix/main.ts:13]: wdym?
imagen

**Review by Keiser04 — COMMENTED** [review]
idk if i can let .5 or no

File: src/pages/list.json [hunk 4]
```diff
@@ -648,5 +648,10 @@
     "domain": "https://witanime.pics",
     "type": "anime",
     "name": "WitAnime"
+  },
```
Comment (lolamtisch) [inline src/pages/list.json:651]: Please do not commit the list.json

File: src/pages/AnimeFenix/meta.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "search": "*://*.animefenix.tv/animes?q={searchtermPlus}",
```
Comment (lolamtisch) [inline src/pages/AnimeFenix/meta.json:2]: Please use a url without wildcards, as it is to generate an search url and not to match one

File: .gitignore [hunk 6]
```diff
@@ -49,3 +49,4 @@ src/utils/quicklinks.json
 
 # IDE
 .idea
+src/pages/list.json
```
Comment (lolamtisch) [inline .gitignore:52]: Move it to `src/utils/quicklinks.json` for consistence

File: src/pages/AnimeFenix/main.ts [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    getOverviewUrl: url => {
+      const urlParts = url.split('/');
+      if (urlParts[3] === 'ver') {
+        urlParts.splice(3, 1);
+      }
+      urlParts[urlParts.length - 1] = urlParts[urlParts.length - 1].replace(/-\d+(\.\d+)?$/, '');
+      return urlParts.join('/');
+    },
+    getEpisode: url => {
```
Comment (lolamtisch) [inline src/pages/AnimeFenix/main.ts:35]: Episode wrong:
https://www3.animefenix.tv/ver/mato-seihei-no-slave-8#22

File: src/pages/AnimeFenix/main.ts [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  sync: {
+    getTitle: url => {
+      const titleElement = document.querySelector('.hero h1');
+      let title = titleElement?.textContent ? titleElement.textContent.trim() : null;
+
+      if (!title) {
+        const urlTitle = url.split('/')[4];
+        title = urlTitle ? urlTitle.replace(/-/g, ' ').replace(/\b\w/g, l => l.toUpperCase()) : '';
+      }
```
Comment (lolamtisch) [inline src/pages/AnimeFenix/main.ts:20]: This section will still return a wrong title.
image

**Review by lolamtisch — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 6 above)
Comment (Keiser04) [reply .gitignore]: how?

**Comment by Keiser04** [comment]
__

File: src/pages/list.json (hunk 4 above)
Comment (Keiser04) [reply src/pages/list.json]: mb

**Pull request closed without merging**